lib/utils/snack_helper.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+void showErrorMessage(BuildContext context ,{required String message}) {
```

Comment:
make it one function and pass the color in it

Reply:
done